### PR TITLE
feat: add support for Terminal API support

### PR DESCRIPTION
**Pull request by vrialland**
**Description**
This PR adds support of the POS terminal management API (https://docs.adyen.com/api-explorer/#/postfmapi/v1/overview)

**Tested scenarios**
All methods from the API have been tested with mocks as it was done for the other endpoints:
- `/assignTerminals`
- `/findTerminal`
- `/getStoresUnderAccount`
- `/getTerminalDetails`
- `/getTerminalsUnderAccount`

All example JSON data was taken from the API explorer test requests

### CODE REVIEW

File: test/TerminalTest.py [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+import unittest
+
+import Adyen
+
+try:
+    from BaseTest import BaseTest
+except ImportError:
+    from .BaseTest import BaseTest
+
+VERSION = pkg_resources.get_distribution("Adyen").version
```
Comment (vrialland) [inline test/TerminalTest.py:11]: Not sure if I should use the version from the settings or the one from the package installed :thinking:

**Comment by acampos1916** [comment]
Thanks for your contributions @vrialland

**Comment by vrialland** [comment]
Hey @acampos1916, @Aleffio, @AlexandrosMor, @martinsrenato, @rikterbeek, sorry to ping you all, but can I have an update on this PR? This is a feature we need for one of our current development :pray:

**Comment by acampos1916** [comment]
Apologies for the delay @vrialland, LGTM!

**Review by acampos1916 — DISMISSED** [review]

**Comment by AlexandrosMor** [comment]
fixes #154

**Review by Alex-Pascal — DISMISSED** [review]

**Review by jillingk — APPROVED** [review]

**Pull request merged**